{
  "uid" : "3d032fbc75bfc701",
  "name" : "nonReentrant function can be called",
  "fullName" : "Contract: ReentrancyGuard.nonReentrant function can be called",
  "historyId" : "e59896602ed4a139da28e26adc8442c",
  "time" : {
    "start" : 1635554984417,
    "stop" : 1635554985420,
    "duration" : 1003
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a183cc70>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a183cc70>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/security/ReentrancyGuard.test.js:10:51)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: ReentrancyGuard"
  }, {
    "name" : "testClass",
    "value" : "Contract: ReentrancyGuard"
  }, {
    "name" : "testMethod",
    "value" : "nonReentrant function can be called"
  }, {
    "name" : "package",
    "value" : "Contract: ReentrancyGuard"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "306c0938e4962d0b",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0fb4b20>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554916231,
        "stop" : 1635554917231,
        "duration" : 1000
      }
    }, {
      "uid" : "c7c3a2ae9aedf186",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0670970>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554908440,
        "stop" : 1635554909033,
        "duration" : 593
      }
    }, {
      "uid" : "5accebe5155312b9",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a07c9760>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554897560,
        "stop" : 1635554898150,
        "duration" : 590
      }
    }, {
      "uid" : "c1ce417fe78f72de",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a068e400>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554892999,
        "stop" : 1635554893590,
        "duration" : 591
      }
    }, {
      "uid" : "371ab4c6b6ddad12",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a183c9d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554848366,
        "stop" : 1635554849335,
        "duration" : 969
      }
    }, {
      "uid" : "2f4022449b3c4155",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0675580>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554828166,
        "stop" : 1635554828924,
        "duration" : 758
      }
    }, {
      "uid" : "ac07012750e97cef",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0684280>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554815725,
        "stop" : 1635554816318,
        "duration" : 593
      }
    }, {
      "uid" : "e67481339cba08bb",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0679340>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554812889,
        "stop" : 1635554813482,
        "duration" : 593
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "bfc7c1857318205f",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/bfc7c1857318205f",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551254088,
          "stop" : 1635551254158,
          "duration" : 70
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "3d032fbc75bfc701.json",
  "parameterValues" : [ ]
}